ci/prow/check-commits-number

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+#FIXME: check if there is a prow github token that can be use to increase the api rate limit from 60 req/h to 5000 req/h
+
+number_of_commits=$(curl -H 'Accept: application/vnd.github+json' \
+    https://api.github.com/repos/${REPO_OWNER}/${REPO_NAME}/pulls/${PULL_NUMBER} | jq '.commits')
+if [[ ${number_of_commits} != 1 ]]; then
```

Comment:
I think we could use `git ` to do that: `git rev-list --count HEAD ^main`

Reply:
Good point.

Fixed.